src/Microsoft.SqlTools.ServiceLayer/EditData/EditSession.cs

```diff
@@ -465,6 +519,20 @@ public EditUpdateCellResult UpdateCell(long rowId, int columnId, string newValue
             return result;
         }
 
+        /// <summary>
+        /// Gets the column information including name and editability for the associated result set
+        /// </summary>
+        /// <returns>Array of column information</returns>
+        public EditColumnInfo[] GetColumnInfo()
+        {
+            ThrowIfNotInitialized();
+            return associatedResultSet?.Columns?.Select(c => new EditColumnInfo
+            {
+                Name = c.ColumnName,
+                IsEditable = c.IsUpdatable
```

Comment:
Out of curiosity, if the column already has an `isUpdatable` property, why rename to row's prop to `isEditable`?